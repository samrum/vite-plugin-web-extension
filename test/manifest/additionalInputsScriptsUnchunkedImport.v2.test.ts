import { getResourceDir, runManifestV2Test } from "./manifestTestUtils";

const resourceDir = getResourceDir("additionalInputsScriptsUnchunkedImport");

runManifestV2Test("additionalInputsScriptsUnchunkedImport", () => ({}), {
  additionalInputs: {
    scripts: [`${resourceDir}/script1.js`, `${resourceDir}/script2.js`],
  },
});
