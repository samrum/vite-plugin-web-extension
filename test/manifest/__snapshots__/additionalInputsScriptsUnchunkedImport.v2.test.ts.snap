// Vitest Snapshot v1

exports[`additionalInputsScriptsUnchunkedImport - Manifest V2 1`] = `
[
  {
    "fileName": "test/manifest/resources/additionalInputsScriptsUnchunkedImport/script1.js",
    "name": undefined,
    "source": "function log(message) {
  console.log(message);
}
log(\\"script1\\");
",
    "type": "asset",
  },
  {
    "fileName": "test/manifest/resources/additionalInputsScriptsUnchunkedImport/script2.js",
    "name": undefined,
    "source": "console.log(\\"script2\\");
",
    "type": "asset",
  },
  {
    "fileName": "manifest.json",
    "name": undefined,
    "source": "{
  \\"version\\": \\"1.0.0\\",
  \\"name\\": \\"Manifest Name\\",
  \\"manifest_version\\": 2
}",
    "type": "asset",
  },
]
`;
